crypto/schemes_test.go

```diff
@@ -129,3 +154,41 @@ func TestVerifyBeacon(t *testing.T) {
 		require.NoError(t, err)
 	}
 }
+
+func TestGetSchemeByID(t *testing.T) {
+	tests := []struct {
+		name string
+		want bool
+	}{
+		{"", true},
+		{crypto.DefaultSchemeID, true},
+		{crypto.ShortSigSchemeID, true},
+		{crypto.SigsOnG1ID, true},
+		{crypto.UnchainedSchemeID, true},
+		{"nonexistentschemename", false},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name+"byID", func(t *testing.T) {
+			got, gotBool := crypto.GetSchemeByID(tt.name)
+			if gotBool && got.Name != tt.name && tt.name != "" {
+				t.Errorf("GetSchemeByID() got = %v, want %v", got, tt.name)
+			} else if tt.name == "" && got.Name != crypto.DefaultSchemeID {
```

Comment:
I think it's a little confusing to do checks on the name here; it would be clearer to format the test as `{ name string, withoutDefault bool, withDefault bool }`